public/components.css

```diff
@@ -0,0 +1,599 @@
+    border: 3px solid teal;
+    margin-right: 2rem;
+    padding: 1rem;
+    border-radius: 2rem;
+}
+
+.offwhite-card {
+    background-color: #f0e3be;
+    border: 3px solid rgb(0, 0, 0);
+    margin-right: 2rem;
+    padding: 1rem;
+    border-radius: 2rem;
+}
+
+.avatars {
+    text-align: center;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+.card-heading {
+    font-size: 40px;
+    padding-left: 1rem;
+    margin-bottom: 1rem;
+}
+
+.alerts {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+}
+
+.alert {
+    padding: 10px;
+    letter-spacing: 2px;
+    font-weight: bold;
+    padding-left: 1rem;
+    margin-bottom: 10px;
+    color: white;
+    font-size: 25px;
+}
+
+.sub-heading {
+    margin-bottom: 1rem;
+    margin-top: 1rem;
+}
+
+
+/* badges are form header */
+
+.image>div {
+    width: 15px;
+    height: 15px;
+    background-color: green;
+    border-radius: 50%;
+    position: absolute;
+    right: -5%;
+    top: 50%;
+}
+
+.badges-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+}
+
+div.green-dot {
+    background: green;
+}
+
+div.gray-dot {
+    background: gray;
+}
+
+div.red-dot {
+    background: red;
+}
+
+.green-image {
+    background: url("../images/female.png");
+    width: 5rem;
+    height: 5rem;
+    background-position: center;
+    background-repeat: no-repeat;
+    position: relative;
+}
+
+.red-image {
+    background: url("../images/female.png");
+    width: 5rem;
+    height: 5rem;
+    background-position: center;
+    background-repeat: no-repeat;
+    position: relative;
+}
+
+.gray-image {
+    background: url("../images/male.png");
+    width: 5rem;
+    height: 5rem;
+    background-position: center;
+    background-repeat: no-repeat;
+    position: relative;
+}
+
+div.dot {
+    width: 100px;
+    height: 30px;
+    background-color: rgb(250, 250, 250);
+    border-radius: 0px;
+    position: absolute;
+    top: 38%;
+    left: 100%;
+    display: none;
+    border: 1px solid black;
+}
+
+.image:hover>div.dot {
+    display: block;
+    border-radius: 1rem;
+    text-align: center;
+    font-size: 0.7rem;
+    padding-top: 0.4rem;
+}
+
+.icons {
+    border: 1px solid black;
+    padding: 1rem;
+    border-radius: 1rem;
+    font-size: 30px;
+    background-color: white;
+    display: flex;
+    justify-content: space-around;
+    box-shadow: 5px 4px 9px 1px rgb(0 0 0);
+    position: relative;
+}
+
+.icon-tag {
+    position: relative;
+    z-index: 1;
+}
+
+.number-on-icon {
+    position: absolute;
+    height: 20px;
+    width: 20px;
+    border-radius: 50%;
+    border: 0px;
+    z-index: 2;
+    color: white;
+    border: 1px solid white;
+    overflow: hidden;
+    right: -13px;
+    top: -6px;
+}
+
+.inline-button {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 5px;
+    border-radius: 2rem;
+    font-weight: bold;
+}
```

Comment:
add cursor:pointer property on all the buttons

Reply:
Done